packages/plugin-ext/src/main/browser/scm-main.ts

```diff
@@ -258,6 +262,15 @@ export class PluginScmProvider implements ScmProvider {
         this.onDidChangeResourcesEmitter.fire();
     }
 
+    toUrl(uri: vscodeURI | undefined): string | undefined {
+        if (!uri) { return undefined; }
+        if (!uri.path) { return undefined; }
```

Comment:
minor: we can combine the two:

```suggestion
        if (!uri || !uri.path) { return undefined; }
```

Reply:
Makes sense. Done.